src/navigation/components/NavigationDropdownItem/NavigationDropdownItem.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {Fragment, MouseEventHandler} from 'react';
+
+import {NavigationDropdownItem, NavigationItemType} from '../../../models';
+import NavigationItem from '../NavigationItem/NavigationItem';
+import NavigationPopup from '../NavigationPopup/NavigationPopup';
+
+export interface NavigationDropdownProps {
+    data: NavigationDropdownItem;
+    onClick: MouseEventHandler;
+    isActive: boolean;
+    position: number;
+    hidePopup: () => void;
+}
+
+const NavigationDropdown: React.FC<NavigationDropdownProps> = ({
```

Comment:
We agreed that it should be refactored later

Reply:
ok